.github/ffm-audit/audit.py

```
@@ -83,6 +84,68 @@ def main():
+        for typ in re.findall(r"(?:incomplete type|unknown type name|undeclared identifier) '(?:struct )?(\w+)'", slog):
+            if typ in ctype_owner:
+                drop.add(ctype_owner[typ])
+        drop -= sskip
+        if not drop:
+            break
+        sskip |= drop
+    open(os.path.join(work, 'structs.log'), 'w').write(slog)
+    # pair each failure with both numbers: cmp<actual, what OSHI's layout says>
+    sfindings = set()
+    lines = slog.splitlines()
+    for i, l in enumerate(lines):
```

Ruff (0.16.1)

[error] Ambiguous variable name: `l`

(E741)